UPGRADE-2.8.md

```diff
@@ -475,6 +475,8 @@ Security
    }
    ```
 
+ * The `AbstractVoter` is deprecated. Upgrade to `Voter` instead.
+
```

Comment:
Or, we could and I think niw should juste revert the changes made on AbstractVoter for 2.8 entirely, but keep the deprecations. Ie remove the supports method & co. Don't you think ?